README.md

```diff
@@ -3,12 +3,17 @@
 
 | 🛑  IMPORTANT - Early Access 🛑  | 
 | -- |
-| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. The MLAPI package  is on the road to being a fully featured solution. We have solutions architects available on Discord and forums to help you work through issues you encounter. |
+| Boss Room: Small Scale Co-op Sample is built on top of the MLAPI package. 
```

Comment:
why was this removed?

Reply:
Because this is a readme,  i wasn't sure if that was relevant  information for the repo readme.

Reply:
you can put it back, im just not sure it belongs in this readme?